### DISCUSSION TITLE
Require a Type Declaration in SolverJs module

### DISCUSSION THREAD

**Discussion by DarkKnightSafe** [DefinitelyTyped/DefinitelyTyped#57153] [Request a new @types package]
NPM: https://www.npmjs.com/package/solverjs
GitHub: https://github.com/suryapratapsinghsuryavanshi/solverjs

**Comment by martin-badin** [comment]
Hi @DarkKnightSafe  I will create a PR.

**Reply by martin-badin** [reply]
Done #57559
